client/bio-webclient/src/main/kotlin/ac/uk/ebi/biostd/client/integration/web/SubmitClient.kt

```diff
@@ -226,9 +226,15 @@ interface ExtSubmissionOperations {
 }
 
 interface PermissionOperations {
-    fun givePermissionToUser(
+    fun grantCollectionPermission(
         user: String,
-        accessTagName: String,
+        accNo: String,
+        accessType: String,
+    )
+
+    fun grantSubmissionPermission(
```

Comment:
I think we should not mantain two operations but a single one. grantPermision depending if accNo is a collection or a submission I do not think should be relevant. `userPermissionsService.hasPermission` does not care already